flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/utils/FlinkRexUtil.scala

```diff
@@ -196,14 +196,23 @@ object FlinkRexUtil {
     * 4. (a > b OR c < 10) OR b < a -> a > b OR c < 10
     * 5. a = a, a >= a, a <= a -> true
     * 6. a <> a, a > a, a < a -> false
+    * 7. a = 2020 AND SEARCH(a, [2020, 2021]) -> a = 2020
```

Comment:
does `a = 2020 OR SEARCH(a, [2020, 2021])` can be simplified as `a = 2020`  ?

Reply:
Obviously it can't.

Reply:
This should be simplified as `SEARCH(a, [2020, 2021])`.